services/api/src/util.py

```diff
@@ -0,0 +1,16 @@
+import os
+import pickle
+from nnsight.schema.Request import RequestModel
+
+REQUEST_DIR = '/tmp/requests/'
+
+def _save_request(request: RequestModel):
```

Comment:
Dont need to pickle. Can just use json serialized version of RequestModel (RequestModel.model_dump() or .json())

Reply:
Probably should also be put in a separate mongodb collection instead of to disk.

Reply:
Also... we need to purge these. Maybe on COMPLETED purge for that key.